Workbooks/HDInsight/At-Scale/settings.json

```diff
@@ -3,5 +3,5 @@
     "name":"HDInsight At-Scale Insights",
     "author": "Microsoft",
     "isPreview": true,
-    "galleries": [{ "type": "workbook", "resourceType": "Microsoft.HDInsight/Clusters"}, { "type": "workbook", "resourceType": "Azure Monitor", "categoryKey": "azureMonitorInsights", "order": 100 }]
+    "galleries": [{ "type": "hdicluster-insights", "resourceType": "Azure Monitor", "categoryKey": "azureMonitorInsights", "order": 100 }]
```

Comment:
this looks good, can you also include another gallery item for `{ "type": "workbook", "resourceType": "Azure Monitor", "categoryKey": "azureMonitorInsights", "order": 100 }`, so in total there should be two items

Reply:
@yves-chan  added it back. Let me know if it looks good